Guides for Cloudflare Workers

dromzeh opened the pull request:
This pull request introduces guides on configuring Lucia with Hono [+ Drizzle] as well as Itty-Router within a Cloudflare Worker, utilizing D1 as the database, based off `web` and `hono` middleware/documentation.

Also contains instructions for those new to wrangler/workerd: including how to install wrangler, create a worker, deploying to Cloudflare & local development.

pilcrowOnPaper commented:
@dromzeh Hey, thanks for opening a PR!

I had to do do some tweaking since Cloudflare Workers isn't just a framework but a platform that can be used by most major frameworks. I moved all the info that you've added into their own respective page (D1 docs, Prisma docs, session attributes guide, etc). I also removed Drizzle and Hono specific content since most of those were already in the docs.

dromzeh commented:
> @dromzeh Hey, thanks for opening a PR!
> 
> I had to do do some tweaking since Cloudflare Workers isn't just a framework but a platform that can be used by most major frameworks. I moved all the info that you've added into their own respective page (D1 docs, Prisma docs, session attributes guide, etc). I also removed Drizzle and Hono specific content since most of those were already in the docs.

Makes sense, thanks! ^^

pilcrowOnPaper commented:
Big thanks for the PR!
